src/types.ts

```diff
@@ -109,14 +109,14 @@ export interface CookieSerializeOptions {
    */
   secure?: boolean | undefined;
   /**
-   * Specifies the `boolean` value for the [`Partitioned` `Set-Cookie`](rfc-cutler-httpbis-partitioned-cookies)
+   * Specifies the `boolean` value for the {@link https://datatracker.ietf.org/doc/html/draft-cutler-httpbis-partitioned-cookies#section-2.1|`Partitioned` `Set-Cookie`}
```

Comment:
Can you please keep markdown format without @link?

Reply:
Reverted the format 👍